src/common/scheduler/policy/evaluation_result.go

```diff
@@ -0,0 +1,15 @@
+package policy
+
+//EvaluationResult is defined to carry the policy evaluated result.
+//
+//Filed 'Result' is optional.
+//Filed 'Error' is optional
+//
+type EvaluationResult struct {
```

Comment:
But currently in watcher.go around line 60, it's not handling the Result, are you saying it should handle it in future?

The concern is, for example if someone implements his own Policy and return some error in the result send it in channel, it will be useless because the framework doesn't handle it...  So this type is misleading.